lib/bfd.h

```diff
@@ -47,6 +47,11 @@ extern "C" {
 
 #define BFD_PROFILE_NAME_LEN 64
 
+/** To use single hop the hop countmust be set to this. */
+#define BFD_SINGLE_HOP_COUNT 1
+/** To use multi hop the hop count must be at maximum this. */
+#define BFD_MULTI_HOP_MAX_HOP_COUNT 254
```

Comment:
255 ?

Reply:
received bfd packets are at minimum 1 hop away.
so if remote forges a bfd packet with 255, the incoming ttl will be 254.